hw/fpga/scripts/xilinx_generate_clk_wizard.tcl

```diff
@@ -0,0 +1,50 @@
+
+# Define design macros
+set design_name                 xilinx_clk_wizard
+set in_clk_freq_hz              125000000
+set out_clk_freq_hz             20000000
+set clkin1_jitter_ps            80.0
+set clkout1_jitter              172.798
+set clkout1_phase_error         96.948
+set clkout1_requested_out_freq  20
+set mmcm_clkfbout_mult_f        8.000
```

Comment:
do we need this number or we can leave it unset and use the default values? do they change from one Vivado version to the other one?